src/ansys/dpf/post/misc_results.py

```diff
@@ -80,7 +81,10 @@ def _get_result_data_function_of_operator(
         )
 
     def _check_elemental_location(self, **kwargs):
-        """Check if the location keyword with an Elemental value is set. If not, raise Exception."""
+        """Check if the location keyword with an Elemental value is set.
```

Comment:
Elemental should either be lowercase or set off in literal tagging.

Reply:
```suggestion
        """Check if the location keyword with an ``Elemental`` value is set.
```